src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Calendar.cs

```diff
@@ -25,7 +22,7 @@ public CalendarFunction(string functionInvariantName, TexlStrings.StringGetter f
 
         public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
         {
-            yield return new TexlStrings.StringGetter[] { };
+            yield return System.Array.Empty<TexlStrings.StringGetter>();
```

Comment:
>System.Array [](http://example.com/codeflow?start=25&length=12)

[minor] `using System` and only use `Array.Empty<T>` here

Reply:
Good catch!